File: python/vedant_bhatnagar/DevsnestProblems_dailyUpdated/performOperations.py
def solve(n, arr):
    # CODE HERE
    # print(arr)
    x = 0
    for i in range(0, len(arr)):
        if arr[i] == "X++" or arr[i] == "++X":
            x = x+1
            # print(n)
            continue
        if arr[i] == "X--" or arr[i] == "--X":
            x = x-1
            # print(n)
    return x

File: java/ankit_kanyal/devnest_problems/phase_1/PerformOperations.java
// link- https://devsnest.in/algo-challenges/final-value-of-variable-after-performing-operations

static int solve(int n, String[] arr){
//CODE HERE 
int ans=0;
for(String string:arr){
    char start=string.charAt(0);
    char end=string.charAt(string.length()-1);
    if(start=='-'){
        --ans;
    }
    if(end=='-') ans--;
    if(start=='+') ++ans;
    if(end=='+') ans++;
     
}
return ans;
}